Helpers.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Add a toast on the bottom right corner indicating status
+ */
+function toast(title="Message", description="Description"){
+  SpreadsheetApp.getActive().toast(description, title, -1);
+}
+
+
+/**
+ * Dummy function to be able to click the Initialize button (UX feedback that having to double click to activate the features is suboptimal)
+ */
+function initialise(){
+  toast("Initialised","Done")
+}
+
+/**
+ * Sets the trigger to run the tracker every day.
+ *
+ */
+function setDailyTrigger() {
+  deleteTriggers('callPSIAPI');
+  const triggerId = PropertiesService.getDocumentProperties().getProperty('triggerId');
+  if (!triggerId) {
+    const trigger = ScriptApp.newTrigger('callPSIAPI')
+                        .timeBased()
+                        .everyDays(1)
+                        .create();
+    PropertiesService.getDocumentProperties().setProperty(
+        'triggerId', trigger.getUniqueId());
+  }
+  toast("Trigger","Done")
+}
+
+
+
+/**
+ * Sets the trigger to run the CrUX API every day.
+ *
+ */
+function setCrUXDailyTrigger() {
+  deleteTriggers('callCrUX');
+  const triggerId = PropertiesService.getDocumentProperties().getProperty('triggerId');
+  if (!triggerId) {
+    const trigger = ScriptApp.newTrigger('callCrUX')
+                        .timeBased()
+                        .everyDays(1)
+                        .create();
+    PropertiesService.getDocumentProperties().setProperty(
+        'triggerId', trigger.getUniqueId());
+  }
+  toast("Trigger","Done")
```

Comment:
Maybe update this message to differentiate between this trigger and the PSI one above.

Reply:
done!